internal/pkg/agent/cmd/inspect.go

```diff
@@ -417,9 +408,14 @@ func getLogLevel(rawCfg *config.Config, cfgPath string) (logp.Level, error) {
 	return logger.DefaultLogLevel, nil
 }
 
-func printComponents(components []component.Component, streams *cli.IOStreams) error {
+func printComponents(
+	components []component.Component,
+	blocked []component.Component,
```

Comment:
It doesn't look like `blocked` is being used by this function?

Reply:
Ahehe, good catch, fixed 😅 I pushed before my local rebuild was finished to confirm the results.

Tbh it makes me nervous that we have this special `inspect` command with no tests that computes everything through completely different code paths than a live agent... I'd rather we find a way to reuse the `Coordinator` logic which has to do all the same things. But I figured that would be too many big changes for one PR. (Though, we do now have diagnostics, which does fetch most of these same values from a real `Coordinator` via a much more invasive approach).